.github/workflows/qaqc.yaml

```diff
@@ -0,0 +1,57 @@
+name: QualityContol
+
+on:
+  pull_request:
+    paths:
+      - 'book/**'
+      - '.github/workflows/qaqc.yaml'
+    branches:
+      - main
+
+env:
+  DOCKER_IMAGE: uwhackweek/snowex:latest
```

Comment:
it would be good if the image name could be a variable, we could either impose naming restrictions on the github org and repository to make this possible https://github.com/uwhackweek/docker-template/issues/1 or figure out a way to have a single common environment variable file that all the workflows read from. that way a template use knows they only have to edit on thing after 'use this template'